site/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "hacker-news-pwas",
+  "description": "Hacker News readers as Progressive Web Apps",
+  "scripts": {
+    "build": "bundle exec jekyll build",
+    "dev": "bundle exec jekyll serve",
+    "dev-sw": "npm run build && sw-precache --config=sw-precache-config.js && npm run dev",
```

Comment:
Don't think we need to run a build first. `sw-precache --config=sw-precache-config.js` should create the service worker file in the root of the project and `jekyll serve` should build the output folder along with the file.

`sw-precache --config=sw-precache-config.js && npm run dev` should be just fine (but please test it and let me know if that's not the case)